City-of-Helsinki/drupal-hdbt
# UHF-3832 link list external link icon

# [UHF-3832](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3832) External link icon
![image](https://user-images.githubusercontent.com/1191667/152520929-43c7f38f-cbce-4494-a082-f4b9ed3cf552.png)

## What was done
* Added a new hdbt icon for `link-external` that works together in tandem with arrow-right icon
* Moved hds `link-external` to unused icons
* Replaced link list icon hack with this new icon
* Finetuned icon position in body and header-top areas.

## How to test
* Get this branch
* Create a page with list of links and text paragraphs
* Add regular and external links to these
* Check that external links look like in screenshot (as approved by Sauli)
